inlineplz/linters/__init__.py

```diff
@@ -277,13 +277,20 @@ def run_command(command, log_on_fail=False, log_all=False):
     shell = False
     if os.name == 'nt':
         shell = True
+    local_env = {}
+    safe_envvars = [
+        b'PATH', b'PYTHONPATH', b'GOPATH', b'SystemRoot',
```

Comment:
Ah, if this is about security then the gains from doing this are barely worth it. Processes can just look at the environment variables on their parent if they're feeling malicious. If you want to run code without trusting it you must sandbox it / jail it using OS functionality or machine separation.